catkin.options

```diff
@@ -0,0 +1 @@
+-DMY_CMAKE_CXX_FLAGS=--coverage -g -O0
```

Comment:
so could this not move into cmaka/robosherlock.cmake.in?  Furthermore is -O0 really needed? That is supposed to be the default [link](https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html#index-O0). Same question for -g. Does coverage need this?
.